hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -243,6 +243,17 @@ public void forceCloseRocksDB() {
         }
     }
 
+    @Override
+    public void compactRange() {
+        try {
+            // Waits while compaction is performed on the background threads
+            // rocksdb().flush(new FlushOptions())
+            rocksdb().compactRange();
```

Comment:
compactRange will wait compaction finished?

Reply:
yes, it may cost a long time, we can start async task for it through gremlin-job.